randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -297,8 +300,8 @@ Extra - room_id: [6]
   > Door to Nocturnal Shaft
       Any of the following:
           Varia Suit
-          # Wave and Plasma can stop the last 3 X
-          Plasma Beam and Wave Beam and Damage Boosts (Advanced) and Normal Damage ≥ 207
+          # Wave beam can stop the last 3 X: https://youtu.be/DEvvMkPch0A
```

Comment:
whatever video got uploaded here is wrong, its showcasing the left to right path